ee/clickhouse/queries/trends/util.py

```diff
@@ -98,16 +99,22 @@ def get_active_user_params(filter: Filter, entity: Entity, team_id: int) -> Dict
     return params
 
 
-def populate_entity_params(entity: Entity, table_name: str = "") -> Tuple[Dict, Dict]:
+def populate_entity_params(
+    entity: Entity, team_id: int, table_name: str = "", *, with_prop_clauses: bool = False
+) -> Tuple[Dict, Dict]:
     params, content_sql_params = {}, {}
     if entity.type == TREND_FILTER_TYPE_ACTIONS:
         action = entity.get_action()
         action_query, action_params = format_action_filter(action, table_name=table_name)
```

Comment:
Nit: we can simplify this since params is empty coming in here:

```suggestion        
        action_query, params = format_action_filter(action, table_name=table_name)
```